components/cyclebutton/stories/template.js

```diff
@@ -2,20 +2,54 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
 // import { ifDefined } from 'lit-html/directives/if-defined.js';
 
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js"
+
 import '../index.css';
-import '../skin.css';
+import "@spectrum-css/actionbutton/index.css"
+import "@spectrum-css/actionbutton/themes/spectrum.css"
+import "@spectrum-css/actionbutton/themes/express.css"
```

Comment:
```suggestion
```

Leveraging the template above means you don't need to pull in the css assets (they'll come with the template automatically).